rclcpp/include/rclcpp/clock.hpp

```diff
@@ -121,6 +123,18 @@ class Clock
     JumpHandler::post_callback_t post_callback,
     const rcl_jump_threshold_t & threshold);
 
+  /**
+   * Get current time from the time source specified by Clock::get_clock_type() const noexcept.
+   *
+   * \return RCUTILS_RET_OK if time was successfully retrieved
+   * \param now. A time point value pointer that will contain the
```

Comment:
```suggestion
   * \param now A time point value pointer that will contain the
```

Reply:
Also, nitpick: `\param` should come before `\return` which should be before `\throws`

Reply:
Words is a bit off too, maybe this?

> \param now A pointer to time point value, which will be set to the current time according to the underlying clock.